package.json

```diff
@@ -4,26 +4,30 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
+    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js --timeout 45000 --exit",
```

Comment:
Why the `--timeout 45000 --exit`?

Reply:
Initially i had some async test that was timing out. I have refactored my code and it doesn't take this long any more, but I thought other team member might need it.

Reply:
Ok